projects/igniteui-angular/src/lib/paginator/paginator.component.ts

```diff
@@ -111,20 +111,19 @@ export class IgxPaginatorComponent extends DisplayDensityBase {
     private _overlaySettings: OverlaySettings = {};
     private defaultSelectValues = [5, 10, 15, 25, 50, 100, 500];
 
-    /**
-     * Sets the class of the IgxPaginatorComponent based
-     * on the provided displayDensity.
-     */
+    /** @hidden @internal **/
     @HostBinding('class.igx-paginator--cosy')
     public get classCosy(): boolean {
         return this.displayDensity === DisplayDensity.cosy;
     }
 
+    /** @hidden @internal **/
     @HostBinding('class.igx-paginator--compact')
     public get classCompact(): boolean {
         return this.displayDensity === DisplayDensity.compact;
     }
 
+    /** @hidden @internal **/
     @HostBinding('class.igx-paginator')
     public get classComfortable(): boolean {
         return this.displayDensity === DisplayDensity.comfortable;
```

Comment:
Can we directly make these private?

https://github.com/IgniteUI/igniteui-angular/issues/12819

Reply:
Sure I can do these ones, the task remains though :)